suite/tests/client-interface/count-bbs.templatex

```diff
@@ -0,0 +1,5 @@
+(<Application .*simple_app.*AVX-512 was detected at PC 0x[0-9a-f]+. AVX-512 is not fully supported yet.>
```

Comment:
This is not good: a bunch of our tests fail with this message?   I would say just build them w/o 512 rather than all this churn in the output files and w/o having them fail up front and not test what they're supposed to test.

Reply:
No, reverted all of them. This stemmed from the fact that earlier I wasn't checking for !instr_is_meta when checking for EVEX instructions in instr_may_write_zmm_or_opmask_register(). If prefix and raw bits are not set, the function checks the operands leading to a decode. The decoder issues the warning. This is all gone now, all the .expect/.template have been reverted.

Reply:
P.S. none of these tests are built with AVX-512.

Reply:
FTR, accepting fatal errors and enshrining them in golden output was *not* the right solution.  If this error could show up in a random app, this would break all kinds of things.

Reply:
It's not a fatal error. It's a warning. The thought was that we are decoding the meta code, hence the warning would be ok. But it was not, see above.